Tasks/Andrey Kruk/task_5/task5.py

```diff
@@ -0,0 +1,53 @@
+def check_str(some_string):
+    count_upper = 0
+    count_lower = 0
+    for i in some_string:
+        if i.isalpha() and i == i.upper():
+            count_upper += 1
+        elif i.isalpha() and i == i.lower():
+            count_lower += 1
```

Comment:
the isalpha check would look better as a separate one, in yore case this check works twice if the **if** doesn't trigger its logic